web/src/stream_ui_updates.js

```diff
@@ -286,3 +280,18 @@ export function update_setting_element(sub, setting_name) {
     const $elem = $(`#id_${CSS.escape(setting_name)}`);
     settings_org.discard_property_element_changes($elem, false, sub);
 }
+
+export function enable_or_disable_add_subscribers_elements($container_elem, enable_elem) {
+    const $input_element = $container_elem.find(".input").expectOne();
+    const $add_subscribers_button = $container_elem
+        .find('button[name="add_subscriber"]')
+        .expectOne();
+
+    $input_element.prop("contenteditable", enable_elem);
+    $add_subscribers_button.prop("disabled", !enable_elem);
+
+    if (enable_elem) {
+        $add_subscribers_button.css("pointer-events", "");
```

Comment:
@sahil839 I have updated the PR to show the "not-allowed" cursor for disabled field and disabled input and changed the position of tooltip for "Add all users" button.
 Regarding CSS properties changed through Js  `initialize_disable_btn_hint_popover` is called at around 5 places in code. So for doing changes in this function we need to make changes in all those places. So, should I do the changes inside the function ?

Reply:
We show "not-allowed" cursor in the space between the input and "Add" button as well. Not sure how important it is as there is very less space. 

I think we can do the "pointer-events" change as follow-up, since it was present before this PR as well.